### PR TITLE
Add insert tab to toolbar

### PR DESCRIPTION
**Pull request by marthacryan**
Adds a tab for "Insert" to the toolbar that looks like this: 
image

Todo:
- [x] Add special behavior for "Anti-Merge"
- [x] Add special behavior for 3 special graph types

### CODE REVIEW

File: mitosheet/src/mito/components/toolbar/InsertTabContents.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        sheetIndex: number,
+        closeOpenEditingPopups: () => void
+    }): JSX.Element => {
+
+    return (<div className='mito-toolbar-bottom'>
+        <ToolbarButton
+            action={props.actions.buildTimeActions[ActionEnum.Pivot]}
+            setEditorState={props.setEditorState}
+        />
+        <div className='mito-pivot-buttons'>
```
Comment (naterush) [inline mitosheet/src/mito/components/toolbar/InsertTabContents.tsx:35]: I don't see this class anywhere, and I missing it?

File: mitosheet/src/mito/types.tsx [hunk 2]
```diff
@@ -964,6 +967,7 @@ export enum ActionEnum {
     Rename_Column = 'rename column',
     Rename_Dataframe = 'rename dataframe',
     Rename_Graph = 'rename graph',
+    Scatter_Plot = 'scatter graph',
```
Comment (naterush) [inline mitosheet/src/mito/types.tsx:970]: I would be down to rename these actions to GRAPH_BAR, GRAPH_LINE, etc -- just so alphabetically they are close.

Comment (naterush) [reply mitosheet/src/mito/types.tsx:970]: And moving in the actions.tsx file.

**Review by naterush — CHANGES_REQUESTED** [review]
Minor code cleanup, but generally looks pretty good.

**Comment by naterush** [comment]
Screen Shot 2023-11-01 at 9 58 13 AM

I think you maybe forgot to push some CSS!

**Comment by marthacryan** [comment]
> Screen Shot 2023-11-01 at 9 58 13 AM
> I think you maybe forgot to push some CSS!

Oh weird I think this is a safari vs chrome thing. I'll fix it!

**Comment by naterush** [comment]
I think we could have a bit more horizontal space between the transpose button and the divider. Let's make the divider just always handle spacing itself, with some margins?
Screen Shot 2023-11-01 at 10 57 35 AM

**Comment by naterush** [comment]
Noting that icons might need some tweaks in the future (merge and concat aren't super legible), but we can fix that up late.

**Pull request merged**